tests/prelude/Prelude.purs

```diff
@@ -556,6 +558,8 @@ module Prelude.Unsafe where
 
   foreign import unsafeIndex :: forall a. Array a -> Number -> a
 
+  foreign import reflectParameterName :: (Int -> String) -> String
```

Comment:
Can we make this `(a -> b) -> String`? Seems like it might be useful for other tests one day.